src/Symfony/Component/Serializer/Context/DefaultContext.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Symfony\Component\Serializer\Context;
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+class DefaultContext implements ContextInterface
```

Comment:
`final`,  but IMHO we should get read of this class and use an `array` or an `iterable` instead.